Dockerfile

```diff
@@ -1,8 +1,27 @@
 ARG ARCH="amd64"
 ARG OS="linux"
+FROM quay.io/prometheus/golang-builder:main as builder
+WORKDIR /workspace
+
+# Copy the required source files
+COPY .git/ .git/
```

Comment:
I was wondering if we could COPY . since it's a multi stage we will only get what we need in the end, using COPY might make our lives a bit easier, did you see any problem on that @jan--f ?

Reply:
:+1: 
I was mostly not sure if `COPY . .` was a thing...but TIL :)